packages/design-system/src/index.ts

```diff
@@ -100,6 +105,10 @@ export {
 	StatusWarning,
 	StatusSuccessful,
 	StatusFailed,
+	SkeletonParagraph,
+	SkeletonHeading,
+	SkeletonButton,
+	SkeletonButtonIcon,
```

Comment:
Do you really hate me? I tried to keep them alpha sorted! 🤣

Reply:
Haha ok I'll try to as well!